src/main/java/com/example/bean/FileUntrustedParent.java

```diff
@@ -0,0 +1,70 @@
+package com.example.bean;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Calendar;
+import java.util.Date;
+
+public class FileUntrustedParent {
+
+	String username;
+
+	public FileUntrustedParent() {
+		System.out.println("ConstructorParent");
+
+		// Add 10 minutes to date
+		Calendar now = Calendar.getInstance();
+		now.add(Calendar.MINUTE, 10);
+		Date teenMinutesFromNow = now.getTime();
+
+		try {
+			Runtime rt = Runtime.getRuntime();
+
+			String[] command = new String[] { "sudo", "date", "-s", teenMinutesFromNow.toString() };
+
+			// Not working: String command = "sudo date -s \'" + teenMinutesFromNow.toString() + "\"";
+			System.out.println(command);
+			Process proc = rt.exec(command);
+
+			BufferedReader stdInput = new BufferedReader(new InputStreamReader(proc.getInputStream()));
+
+			BufferedReader stdError = new BufferedReader(new InputStreamReader(proc.getErrorStream()));
+
+			// read the output from the command
+			System.out.println("Here is the standard output of the command:\n");
+			String s = null;
+			while ((s = stdInput.readLine()) != null) {
+				System.out.println(s);
+			}
+
+			// read any errors from the attempted command
+			System.out.println("Here is the standard error of the command (if any):\n");
+			while ((s = stdError.readLine()) != null) {
+				System.out.println(s);
+			}
+			int exitVal = proc.waitFor();
+
+			System.out.println("Process exitValue: " + exitVal);
+
+		}
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- 14131253a5f11bef3940941a19c5f354 -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - Improper Resource Shutdown or Release
_High CWE-404_
Leak of a system resource

The system resource will not be reclaimed and reused, reducing the future availability of the resource.

### How to fix
The application must shut down or close any opened resource (such as a database connection, file handle, or input/output stream) after it is finished using that resource. The implementation should account for all possible execution paths where use of a resource ceases, including when exceptions occur.

Where possible, it is recommended to use the dispose pattern provided by the language or framework in question, e.g., the "using" statement in C# or the "try-with-resources" statement in Java to ensure a disposable or closeable object is disposed or closed on all paths exiting a block, including exception cases. Otherwise, calling "Dispose" (C#) or "close" (Java) in a "finally" block is equally effective but more verbose and prone to mistakes.